integration_test.go

```diff
@@ -3,9 +3,6 @@
 // Use of this source code is governed by the Apache License 2.0
 // that can be found in the COPYING file.
 
-//go:build integration
```

Comment:
Any reason why you are removing the build tag?

Reply:
Yeah sorry, I accidentally removed the tags and forgot to add back 
Fixed https://github.com/tsuna/gohbase/pull/257/files#diff-ed1088fda4ca38792b99302b355705eb525017fca294e83858422bb430d8cc59